src/Search/Search.Tests/Tests/CustomAnalyzerTests.cs

```diff
@@ -273,7 +273,7 @@ public void CanCreateAllAnalysisComponents()
                         tokenChars: new[] { TokenCharacterKind.Letter }),
                     new NGramTokenizer(SearchTestUtilities.GenerateName(), minGram: 2, maxGram: 4, tokenChars: new[] { TokenCharacterKind.Letter }),
                     new ClassicTokenizer(SearchTestUtilities.GenerateName(), maxTokenLength: 100),
-                    new KeywordTokenizer(SearchTestUtilities.GenerateName(), bufferSize: 100),
```

Comment:
>KeywordTokenizer [](start = 24, length = 16)

Seems KeywordTokenizer will no longer be tested. Should'nt these be left for regression purpose, until the entire class has been deleted.